config/webpack.common.config.js

```diff
@@ -129,14 +122,16 @@ const commonWebpackConfig = {
         }, {
           from: RESOURCES_PATH + 'i18n/',
           to: './resources/i18n/'
-        }, {
-          from: RESOURCES_PATH + 'img/',
-          to: './resources/img/'
-        }, {
-          from: RESOURCES_PATH + 'help/',
-          to: './resources/help/'
         }
-    ]),
+        // , {
+        //   from: RESOURCES_PATH + 'img/',
+        //   to: './resources/img/'
+        // }, {
+        //   from: RESOURCES_PATH + 'help/',
+        //   to: './resources/help/'
+        // }
```

Comment:
You can remove the commented code

Reply:
I leave it in for the moment, mounts will be possibly reintroduced soon.